api-irpf/README.md

```diff
@@ -6,4 +6,6 @@ Api de importação e leitura das notas de corretagem.
 
 2. (Alternativa) `mvn spring-boot:run -Drun.profiles=dev`
 
-3. Após start da aplicação é possível acessar pelo link: [http://localhost:8085/](http://localhost:8085/)
\ No newline at end of file
+3. Após start da aplicação é possível acessar pelo link: [http://localhost:8085/](http://localhost:8085/)
+
+4. Ideia para o domínio dos dados extraídos: [https://miro.com/app/board/o9J_loak7CA=/?invite_link_id=785499274109](https://miro.com/app/board/o9J_loak7CA=/?invite_link_id=785499274109)
```

Comment:
Já que o domínio já foi criado, acho que podemos tirar esse link do README.

Reply:
Acho que pode deixar pq, esse material vai fazer parte da documentação, assim como outros materiais que a gente for gerando.